<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>BlueTooth</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QListWidget" name="list"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_info">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Bluetooth&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_disconnect">
         <property name="text">
          <string>断开</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_openBluetooth">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_scan">
         <property name="text">
          <string>扫描</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_closeDevice">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_send">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_clear">
         <property name="text">
          <string>清除</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="checkBox_discoverable">
         <property name="text">
          <string>可见性</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Console</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="4">
        <widget class="QPushButton" name="pushButton_fr">
         <property name="text">
          <string>对角</string>
         </property>
        </widget>
       </item>
       <item row="12" column="3">
        <widget class="QPushButton" name="pushButtonTurnMode">
         <property name="text">
          <string>平移</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="pushButtonRight">
         <property name="text">
          <string>右转</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="pushButtonForward">
         <property name="text">
          <string>前进</string>
         </property>
        </widget>
       </item>
       <item row="19" column="2">
        <widget class="QLineEdit" name="lineEdit_16"/>
       </item>
       <item row="8" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="16" column="2">
        <widget class="QLineEdit" name="lineEdit_10"/>
       </item>
       <item row="18" column="2">
        <widget class="QLineEdit" name="lineEdit_13"/>
       </item>
       <item row="5" column="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="lineEdit_9"/>
       </item>
       <item row="8" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="4">
        <widget class="QPushButton" name="pushButtonMode">
         <property name="text">
          <string>遥控模式</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="pushButtonLeft">
         <property name="text">
          <string>左转</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_7"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEdit_8"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item row="9" column="4">
        <widget class="QPushButton" name="pushButton_br">
         <property name="text">
          <string>对角</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="8" column="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item row="16" column="4">
        <widget class="QLineEdit" name="lineEdit_12"/>
       </item>
       <item row="10" column="3">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
       <item row="16" column="3">
        <widget class="QLineEdit" name="lineEdit_11"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item row="19" column="3">
        <widget class="QLineEdit" name="lineEdit_17"/>
       </item>
       <item row="19" column="4">
        <widget class="QLineEdit" name="lineEdit_18"/>
       </item>
       <item row="14" column="3">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="pushButton_fl">
         <property name="text">
          <string>对角</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="pushButtonBackward">
         <property name="text">
          <string>后退</string>
         </property>
        </widget>
       </item>
       <item row="18" column="3">
        <widget class="QLineEdit" name="lineEdit_14"/>
       </item>
       <item row="18" column="4">
        <widget class="QLineEdit" name="lineEdit_15"/>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEdit_1"/>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="pushButton_bl">
         <property name="text">
          <string>对角</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QSlider" name="horizontalSliderSpeed">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>speed:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
